src/pages/dogs/[id].jsx

```diff
@@ -75,6 +79,36 @@ export default function IndividualDogPage() {
   return (
     // Artificial spacing until nav is created
     <div className={`container mx-auto order-b border-gray-300`}>
+      <Toaster />
+      {showLogModal ? (
+        <>
+          <LogModal
+            dogId={ dog._id }
+            userId={ dog.instructors[0]._id }
+            onClose={() => {
+              setShowLogModal(false);
+            }}
+            onSubmit={(success) => {
+              // TODO toast animation
+              if (success) {
+                toast.custom((t) => (
+                  <div className={`h-12 px-6 py-4 rounded shadow border border-ca-pink-shade justify-center items-center inline-flex bg-ca-pink text-white text-lg font-normal
+                    ${t.visible ? 'animate-enter' : 'animate-leave'}`}
+                  >
+                    <span className="font-bold">New log</span>&nbsp;<span>was successfully added.</span>
```

Comment:
Yeah since the toast is only shown if there is a server error and the user doesn't need to know the specifics about that. The other error handling is caught before trying to save so that's communicated in the modal instead of the toast